notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11655,6 +11655,50 @@ giant turtle [Gamera.]
 giant dinosaur [The Gyaos.]
 
 
+movie: Wing Commander (1999)
+============================
+
+:: Title
+Wing Commander
+
+:: Description
+The year is 2654, and it is up to a crop of fresh Terran Confederation
+military recruits to thwart the efforts of the cat-like Kilrathi aliens from
+exterminating the human race. It is loosely based on the video game series of
```

Comment:
A confusing change of vantage point in these two sentences. (Just saying)

Reply:
I agree. How about this:

"Set in the year 2654, the story follows a crop of fresh Terran Confederation military recruits in their effort to thwart a hostile alien empire from exterminating the human race. It is loosely based on the video game series of the same name."